.github/actions/spelling/excludes.txt

```diff
@@ -54,14 +54,26 @@ SUMS$
 \.zip$
 ^consolegit2gitfilters\.json$
 ^dep/
-^oss/
+^doc/reference/master-sequence-list.csv$
 ^doc/reference/UTF8-torture-test\.txt$
+^oss/
+^src/host/ft_uia/run\.bat$
+^src/host/runft\.bat$
+^src/host/runut\.bat$
 ^src/interactivity/onecore/BgfxEngine\.
 ^src/renderer/wddmcon/WddmConRenderer\.
+^src/terminal/adapter/ut_adapter/run\.bat$
+^src/terminal/parser/delfuzzpayload\.bat$
+^src/terminal/parser/ft_fuzzer/run\.bat$
```

Comment:
* Warning: src/terminal/adapter/ut_adapter/run.bat: line 1, columns 1-1, Warning - Skipping `src/terminal/adapter/ut_adapter/run.bat` because there seems to be more noise (4) than unique words (2) (total: 4 / 2). (noisy-file)
* Warning: src/terminal/parser/delfuzzpayload.bat: line 1, columns 1-1, Warning - Skipping `src/terminal/parser/delfuzzpayload.bat` because there seems to be more noise (4) than unique words (0) (total: 4 / 0). (noisy-file)
* Warning: src/terminal/parser/ft_fuzzer/run.bat: line 1, columns 1-1, Warning - Skipping `src/terminal/parser/ft_fuzzer/run.bat` because there seems to be more noise (4) than unique words (1) (total: 4 / 1). (noisy-file)